.github/workflows/deploy-eb.yml

```diff
@@ -97,12 +99,22 @@ jobs:
           sed -i -e "s/@DD_API_KEY/$DD_API_KEY/g" -e "s/@DD_ENV/$DD_ENV/g" .ebextensions/99datadog-amazon-linux-2.config
           zip -r "$IMAGE_TAG.zip" .ebextensions Dockerrun.aws.json
 
-      - name: Copy to S3
+      - name: Copy Image to S3
         env:
           BUCKET_NAME: ${{ secrets.BUCKET_NAME }}
         run: |
           aws s3 cp $IMAGE_TAG.zip s3://$BUCKET_NAME/$IMAGE_TAG.zip
 
+      - name: Copy Static Assets from Docker Image and Push to S3
+        env:
+          S3_STATIC_ASSETS_BUCKET_NAME: ${{ secrets.S3_STATIC_ASSETS_BUCKET_NAME }}
+          ECR_REPOSITORY: ${{ secrets.ECR_REPO }}
+        run: |
+          mkdir -p /tmp/s3static
+          chmod 777 /tmp/s3static
+          docker run --rm -v /tmp/s3static:/tmp/s3static $ECR_REPOSITORY:$IMAGE_TAG cp -r ./dist/frontend/. /tmp/s3static
+          aws s3 sync /tmp/s3static s3://$S3_STATIC_ASSETS_BUCKET_NAME
```

Comment:
Since the assets have hash in their names and are therefore immutable, we should give them a long expiry (10 years?) with a cli arg `--cache-control max-age=315360000`

Reply:
KIV for now, since there are non-immutable assets which don't have hash (e.g. static images)